cl/opinion_page/templates/includes/rd_metadata_headers.html

```diff
@@ -1,48 +1,47 @@
+    <span class="gray">&mdash;</span>
+  {% endif %}
+  Document
+  <a href="{{ rd.docket_entry.docket.get_absolute_url }}?entry_gte={{ rd.document_number }}#entry-{{ rd.document_number }}">
+    #{{ rd.document_number }}</a>{% if rd.document_type == rd.ATTACHMENT %},
+    Attachment #{{ rd.attachment_number }}{% endif %}
     {% if attachments %}
-      <button
-        class="btn btn-default btn-lg dropdown-toggle"
-        type="button"
-        data-toggle="dropdown"
-        aria-expanded="false"
-        style="vertical-align:bottom; padding:1px 10px; border-radius:3px;"
-      >
-        <span class="caret"></span>
-      </button>
+      <div class="btn-group">
+        <button
+          class="btn btn-default btn-lg dropdown-toggle"
+          type="button"
+          data-toggle="dropdown"
+          aria-expanded="false"
+          style="vertical-align:bottom; padding:1px 10px; border-radius:3px;"
+        >
+          <span class="caret"></span>
+        </button>
 
-      <ul class="dropdown-menu">
-        {% for doc in attachments %}
-          <li {% if doc.attachment_number == rd.attachment_number or not doc.url %}class="disabled"{% endif %}>
-            {% if not doc.url %}
-              <span style="font-size:21px; font-weight:500; padding:3px 18px;">{{ doc.description }}</span>
-            {% else %}
-              <a
-                href="{% if doc.attachment_number != rd.attachment_number %}{{ doc.url }}{% else %}#{% endif %}"
-                style="font-size:21px; font-weight:500; padding:3px 18px;"
-                title="{% if doc.description %}{{ doc.description }}{% elif doc.attachment_number%}Attachment #{{ doc.attachment_number }}{% else %}Document #{{ rd.document_number }}{% endif %}"
-              >
-                {% if doc.attachment_number %}#{{ doc.attachment_number }}{% else %}Document #{{ rd.document_number }}{% endif %}
-                {% if doc.description %}
-                  <span class="gray">&mdash;</span>
-                  {{ doc.description }}
-                {% endif %}
-              </a>
-            {% endif %}
-          </li>
-        {% endfor %}
-      </ul>
-    {% endif %}
-  </h3>
-</div>
+        <ul class="dropdown-menu pull-right medium">
+          {% for doc in attachments %}
+            <li {% if doc.attachment_number == rd.attachment_number or not doc.url %}class="disabled"{% endif %}>
+              {% if not doc.url %}
+                {{ doc.description }}
+              {% else %}
+                <a
+                  href="{% if doc.attachment_number != rd.attachment_number %}{{ doc.url }}{% else %}#{% endif %}"
+                  title="{% if doc.description %}{{ doc.description }}{% elif doc.attachment_number%}Attachment #{{ doc.attachment_number }}{% else %}Document #{{ rd.document_number }}{% endif %}"
+                >
+                  {% if doc.attachment_number %}Att. {{ doc.attachment_number }}{% else %}Entry {{ rd.document_number }}{% endif %}
```

Comment:
Is there a reason to use "Entry" rather than "Doc."?  I think of a docket entry as encompassing the main document and all attachments, so it seems odd.  If I didn't know, would it be unreasonable to think that might take me back to the docket entry?  And while "Attachment" gets abbreviated to "Att.", "Document" becomes "Entry", which is not what I would expect.

Reply:
I've found that Entry is usually the word folks use to refer to these. They say "Entry 22" or "Entry 22-2" etc. Saying "Document 22" is something we've done for a while, but I think it's not what folks use in practice.